pipeline/inferrer/feature_inferrer/Dockerfile

```diff
@@ -1,6 +1,14 @@
-FROM tiangolo/uvicorn-gunicorn-fastapi:python3.7
+FROM python:3.7-slim
```

Comment:
Advantages of this:
- smaller image
- more explicit control
- pinned version of fastapi (for some reason it wasn't pinned in the tiangolo image)
- all dependencies in requirements.txt make local development easier

Reply:
sounds like a very good move to me